src/ert/scheduler/lsf_driver.py

```diff
@@ -24,10 +24,13 @@
 from pydantic import BaseModel, Field
 from typing_extensions import Annotated
 
-from ert.scheduler.driver import Driver
+from ert.scheduler.driver import SIGNAL_OFFSET, Driver
 from ert.scheduler.event import Event, FinishedEvent, StartedEvent
 
 _POLL_PERIOD = 2.0  # seconds
+LSF_FAILED_JOB: Annotated[int, "Return code we use when lsf reports failed jobs"] = (
+    SIGNAL_OFFSET + 32
```

Comment:
Is this considered a signal or is it a returncode?

According to the relevant manpage[^1], signal `32` is the first "real-time" user-defined signal (`SIGRTMIN`), but `glibc` uses the first two for pthreads. This means that `import signal; signal.SIGRTMIN` is `34` on my machine. So, is LSF raising a signal (that is actually used by pthread) or is this unrelated and is just the returncode?

[^1]: https://www.man7.org/linux/man-pages/man7/signal.7.html